examples/ios_app/test/fixtures/bwb.xcodeproj/project.pbxproj

```diff
@@ -1796,7 +1801,7 @@
 				ENABLE_TESTING_SEARCH_PATHS = YES;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(PROJECT_DIR)/third_party",
-					"$(PROJECT_DIR)/bazel-ios_app/external/examples_ios_app_external",
+					"$(BAZEL_EXTERNAL)/examples_ios_app_external",
```

Comment:
Besides for being shorter, I like that these are more readable, especially in diffs.